streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsMetricsImpl.java

```diff
@@ -112,12 +116,19 @@ private String sensorName(String operationName, String entityName) {
      * @throws IllegalArgumentException if tags is not constructed in key-value pairs
      */
     @Override
-    public Sensor addLatencyAndThroughputSensor(String scopeName, String entityName, String operationName, Sensor.RecordingLevel recordingLevel, String... tags) {
-        Map<String, String> tagMap = tagMap(tags);
+    public Sensor addLatencyAndThroughputSensor(String scopeName, String entityName, String operationName,
+                                                Sensor.RecordingLevel recordingLevel, String... tags) {
+        List<String> updatedTagList = new ArrayList(Arrays.asList(tags));
+        updatedTagList.add(scopeName + "-id");
+        updatedTagList.add(entityName);
+        Map<String, String> tagMap = tagMap(updatedTagList.toArray(new String[updatedTagList.size()]));
```

Comment:
Unfortunately I need to add the original tags as well, not just those two lines.